cmd/main.go

```diff
@@ -21,14 +21,17 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/kubernetes-sigs/aws-efs-csi-driver/pkg/driver"
 	"k8s.io/klog"
+
+	"github.com/kubernetes-sigs/aws-efs-csi-driver/pkg/driver"
 )
 
 func main() {
 	var (
-		endpoint = flag.String("endpoint", "unix://tmp/csi.sock", "CSI Endpoint")
-		version  = flag.Bool("version", false, "Print the version and exit")
+		endpoint                = flag.String("endpoint", "unix://tmp/csi.sock", "CSI Endpoint")
+		version                 = flag.Bool("version", false, "Print the version and exit")
+		efsUtilsCfgDirPath      = flag.String("efs-utils-config-dir-path", "/etc/amazon/efs/", "The path to efs-utils config directory")
```

Comment:
hmmm are these flags needed, would a user ever want to change them?

Reply:
I was expecting users would like to change them when running the driver locally for testing (in order to do that we should not talk to IMDS always, instead passing instanceId and region from a flag).

Reply:
efs-utils will always read from /etc/amazon/efs, I think this is just going to confuse people because changing it doesn't actually do anything. All that happens is the driver will copy the files from static path to a different path. But I only know that from looking at the code, a user is not going to know or care